lib/locales/en-US.json

```diff
@@ -37,6 +37,11 @@
       "msSpinner": {
         "loading": "Loading..."
       },
+      "msCodeValidationModal": {
```

Comment:
it should not be called like this, since:
1. this is not a modal
2. this is not the name of the component in /components => should be the same